exampleKnitFlat.txt

```diff
@@ -0,0 +1,24 @@
+k1 yo k2tog k1 k1 k1 k1 k1 k1 k1 k1 k1
```

Comment:
It might be clearer to put these into their own directory `examples/`, so that a dev will know they're not needed for the test suite or for the app to run.